### ISSUE TITLE
Downloading causing issues with TOC

### ISSUE THREAD

**Issue by Rajvir8** [KotatsuApp/kotatsu-parsers#890]
### Source information

Webtoons English

### Steps to reproduce

I downloaded all of Kubera's Chapters starting from 550, and previously had downloaded the next 10 chapters at an earlier point.

Somehow Chapters 559-560 were missing and while browsing online it would repeatedly skip past those chapters and start causing issues with the TOC, it was only after deleting all unread chapters, that they showed up as read and skipped past.

Possible issue with download next 10 chapters combined with Download rest of the chapters a few chapters in?

Redownloaded the chapters manually, and was able to start reading normally again.

Tried searching up downloads and didn't see any active reports on this issue, but thought I'd report it even if I figured out how to fix it for myself.

### Kotatsu version

7.2

### Android version

12

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
